Gemfile

```diff
@@ -11,38 +13,51 @@ def location_for(place, fake_version = nil)
 end
 
 group :system_tests do
-  gem 'rake'
-  gem 'beaker-rspec', *location_for(ENV['BEAKER_RSPEC_VERSION'] || '~> 6.0')
-  gem 'beaker-pe', *location_for(ENV['BEAKER_PE_VERSION'] || '~> 1.1')
-  gem 'listen', '<3.1.0'
+  gem 'rake',                    '~> 0'
+  gem 'beaker-rspec',            *location_for(ENV['BEAKER_RSPEC_VERSION'] || '<  5.7') if RUBY_VERSION <  '2.1.5'
```

Comment:
FYI `RUBY_VERSION` is a string and is compared that way, so be careful.

You're better off with something that always compares the way you would expect like:

```ruby
if Gem::Version.new(RUBY_VERSION.dup) < Gem::Version.new('2.1.5')
```

See https://github.com/puppetlabs/puppet/blob/master/.gemspec#L50 for instance.